cloud_experts_tutorials/rosa-mobb-prerequisites-tutorial.adoc

```diff
@@ -127,7 +127,7 @@ $ aws sts get-caller-identity
 +
 You should receive output similar to the following:
 +
-.Sample output
+.Example output
```

Comment:
I am unable to see preview for this assembly. I can see see similar comment on Line 20 of this file, "//This file is not being built as of 2023-09-22 based on a conversation with Michael McNeill."

Is this expected?

Reply:
Yes, some content is hidden on purpose. I still want to make the fixes.